source/core/src/main/com/csse3200/game/areas/ForestGameArea.java

```diff
@@ -79,6 +122,29 @@ public void create() {
     // Obstacles
     spawnTrees();
 
+
+    // Enemies
```

Comment:
None of this should be here - in fact this entire file doesn't need to be changed from what it is in main.

Reply:
Resolved